x-pack/test/api_integration/deployment_agnostic/apis/observability/onboarding/get_privileges.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { type DeploymentAgnosticFtrProviderContext } from '../../../ftr_provider_context';
+import { type SupertestWithRoleScopeType } from '../../../services';
+
+export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const roleScopedSupertest = getService('roleScopedSupertest');
+
+  let viewerClient: SupertestWithRoleScopeType;
+  let adminClient: SupertestWithRoleScopeType;
+
+  describe('Api Key privileges check', () => {
+    before(async () => {
+      viewerClient = await roleScopedSupertest.getSupertestWithRoleScope('viewer', {
+        withInternalHeaders: true,
+      });
+      adminClient = await roleScopedSupertest.getSupertestWithRoleScope('admin', {
+        withInternalHeaders: true,
+      });
+    });
```

Comment:
we need to invalidate api keys after tests executed:


```
    after(async () => {
      viewerClient.destroy();
      adminClient.destroy();
    });
```

Reply:
Thanks! Renamed the variables and added `after`